docs/sql/oracle/create_schema.sql

```diff
@@ -399,11 +399,13 @@ CREATE UNIQUE INDEX PA_APPLICATION_NAME ON PA_APPLICATION(NAME);
 
 CREATE INDEX PA_OPERATION_USER ON PA_OPERATION(USER_ID);
 
-CREATE INDEX PA_OPERATION_TS_CREATED_IDX ON PA_OPERATION(TIMESTAMP_CREATED);
+CREATE INDEX PA_OPERATION_TS_CREATED ON PA_OPERATION(TIMESTAMP_CREATED);
```

Comment:
May we edit existing indexes?

Reply:
Good point, I'll revert the change and leave the inconsistency. We can deal with it later when introducing Liquibase.